meetuppizza/tests.py

```diff
@@ -32,40 +81,43 @@ def test_signed_in_user_email_displayed_on_home_page(self):
     self.assertContains(response, "bjorn@bjorn.com")
 
   def test_meetup_is_displayed_on_landing_page(self):
```

Comment:
I'll use this as an example of a full-framework request spec vs. one using the [`RequestFactory`](https://docs.djangoproject.com/en/1.9/topics/testing/advanced/#the-request-factory). Right now, it's:

```py
def test_meetup_is_displayed_on_landing_page(self):
  meetup = Meetup.objects.create(name='new meetup', meetup_link='http://www.meetup.com/papers-we-love/')
  response = self.client.get('/')
  self.assertContains(response, "The Lexington")
```

A better isolated test using the `RequestFactory` might look like:

```py
from django.test import RequestFactory
from meetuppizza.views import index # Import your view, since it's just a function.

def test_meetup_is_displayed_on_landing_page(self):
  meetup = Meetup.objects.create(name='new meetup', meetup_link='http://www.meetup.com/papers-we-love/')
  request = RequestFactory().get('/') # Construct a request object without spinning up a full server.
  response = index(request) # Call your view function with the request and get back a response. It's Just A Function.
  self.assertContains(response, "The Lexington")
```